<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>240</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>410</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFolder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Select input Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectWatermark">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Select watermark</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectFolderLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Select Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="watermarkPreview">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>125</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="selectOutputFolder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectOutputFolderLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Select Output Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="watermarkPathLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Watermerk pad</string>
    </property>
   </widget>
   <widget class="QLabel" name="watermarkPath">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>610</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
